homeassistant/components/openuv/__init__.py

```diff
@@ -166,14 +166,17 @@ async def async_update(self) -> None:
 class OpenUvEntity(Entity):
     """Define a generic OpenUV entity."""
 
-    def __init__(self, openuv: OpenUV, sensor_type: str) -> None:
+    def __init__(self, openuv: OpenUV, description: EntityDescription) -> None:
         """Initialize."""
+        self._sensor_type = description.key
```

Comment:
Instead of creating an additional variable, replace the use of `self._sensor_type` in this class with `self.entity_description.key`. That way, this can be removed.